### ISSUE TITLE
MobSF Static Analysis fails on apks upload via api

### ISSUE THREAD

**Issue by ice1080** [MobSF/Mobile-Security-Framework-MobSF#2043]
## ENVIRONMENT

```
OS and Version:
Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-48-generic x86_64)

Python Version:
Python 3.10.6

MobSF Version:
VERSION = '3.5.2'
```

## EXPLANATION OF THE ISSUE

We are using MobSF via a git clone of master, with the latest pulled down. When attempting to upload an apks file via curl to the /scan endpoint, a 500 is returned, with the below error in the logs. 
According to #1728 and the [releases page](https://github.com/MobSF/Mobile-Security-Framework-MobSF/releases), the apks format has been supported since v3.4.6 Beta. Looking through the code changes for that support, it appears that api_static_analysis.py did not get updated to support the addition of .apks files.
I'm mostly just curious if this was intentional or a mistake. 

## STEPS TO REPRODUCE THE ISSUE

```
1. Upload apks file using curl to /api/v1/scan with scan_type=apks
2. View MobSF logs.
```

## LOG FILE

```
  __  __       _    ____  _____       _____  ____  
 |  \/  | ___ | |__/ ___||  ___|_   _|___ / | ___| 
 | |\/| |/ _ \| '_ \___ \| |_  \ \ / / |_ \ |___ \ 
 | |  | | (_) | |_) |__) |  _|  \ V / ___) | ___) |
 |_|  |_|\___/|_.__/____/|_|     \_/ |____(_)____/ 

[INFO] 03/Oct/2022 21:24:08 - ESC[1mESC[34mMobile Security Framework v3.5.2 BetaESC[0m
[INFO] 03/Oct/2022 21:24:08 - OS: Linux
[INFO] 03/Oct/2022 21:24:08 - Platform: Linux-5.15.0-48-generic-x86_64-with-glibc2.35
[INFO] 03/Oct/2022 21:24:08 - Dist: ubuntu 22.04 Jammy Jellyfish
[INFO] 03/Oct/2022 21:24:08 - MobSF Basic Environment Check
[WARNING] 03/Oct/2022 21:24:08 - Dynamic Analysis related functions will not work. 
Make sure a Genymotion Android VM/Android Studio Emulator is running before performing Dynamic Analysis.
[INFO] 03/Oct/2022 21:24:09 - Checking for Update.
[INFO] 03/Oct/2022 21:24:09 - No updates available.
[ERROR] 03/Oct/2022 21:24:20 - Internal Server Error: /api/v1/scan
Traceback (most recent call last):
  File "/opt/Mobile-Security-Framework-MobSF/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/opt/Mobile-Security-Framework-MobSF/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/Mobile-Security-Framework-MobSF/mobsf/MobSF/views/helpers.py", line 92, in wrapper
    return func(*args, **kwargs)
  File "/opt/Mobile-Security-Framework-MobSF/venv/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/opt/Mobile-Security-Framework-MobSF/mobsf/MobSF/views/api/api_static_analysis.py", line 76, in api_scan
    return response
UnboundLocalError: local variable 'response' referenced before assignment
```

**Comment by ajinabraham** [comment]
Hi @ice1080 Are you able to scan it via the UI? If not, can you share the .apks for troubleshooting?

**Comment by ice1080** [comment]
I am able to scan it via the UI, though it doesn't show up in the recent scans page when I do that (not sure if that's intentional or not). Unfortunately, we can't share our .apks at this time. We would also need the api to work outside of the gui, since the scan is occurring in our ci machine. 
Looking at api_static_analysis.py, it seems like it's possible the method `api_scan` might just need to be updated to include apks in the scan type check? Obviously that's an outside perspective and I don't know if that would be all that's needed, just something I noticed based on the stack trace.

**Comment by ice1080** [comment]
I should mention that this isn't a blocking issue for us. In our case we only have one .apk inside our .apks so we're able to update our bundletool to add `--mode=universal`, convert output to zip, unzip, and grab the individual .apk from there for uploading via api (reference - https://stackoverflow.com/questions/53040047/generate-apk-file-from-aab-file-android-app-bundle).
I was mostly just curious if the scan_type of apks was supposed to be supported by the api.

**Comment by ajinabraham** [comment]
I found the root cause, this is fixed in https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/2000/commits/bbf0c0d03ea45e5ee3f1f3f0f972b68416a38580 will be fixed in 3.6.0

**Comment by ajinabraham** [comment]
AAB should be supported now https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/2391
